Docs/Components/rating.html

```diff
@@ -115,29 +115,28 @@ <h3 class="heading-3 comp-heading">Dynamic Rating</h3>
                 <br>
 
                 <div class="comp-box rounded-lg centered-flex-row-container">
-                    <div class="rating-wrapper">
-                        <input type="radio" id="star-5"> 
+                    <div class="star-rating">
+                        <input type="radio" name="rate" id="star-5"> 
                         <label class="fas fa-star star" for="star-5"></label>
-                        <input type="radio" id="star-4"> 
+                        <input type="radio" name="rate" id="star-4"> 
                         <label class="fas fa-star star" for="star-4"></label>
-                        <input type="radio" id="star-3"> 
+                        <input type="radio" name="rate" id="star-3"> 
                         <label class="fas fa-star star" for="star-3"></label>
-                        <input type="radio" id="star-2"> 
+                        <input type="radio" name="rate" id="star-2"> 
                         <label class="fas fa-star star" for="star-2"></label>
-                        <input type="radio" id="star-1"> 
+                        <input type="radio" name="rate" id="star-1"> 
                         <label class="fas fa-star star" for="star-1"></label>
-                    </div>                                                                                                                                                                                                                                      
+                    </div>                                                                                                                                                            
                 </div>
                 <br>
                 <br>
 
                 <div class="code-snippet">
-                    
+                    <script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2FDevansu-Yadav%2FSapphire%2Fblob%2FembedBranch%2Fembed.html%23L502-L513&style=github&showBorder=on&showCopy=on"></script>
```

Comment:
In live preview your rating code snippet is wrong

Reply:
Thanks for pointing this out!